Ginger/Ginger/SolutionWindows/TargetApplicationsPage.xaml.cs

```diff
@@ -187,20 +188,31 @@ private void UpdateApplicationNameChangeInSolution(ApplicationPlatform app)
             foreach (BusinessFlow bf in WorkSpace.Instance.SolutionRepository.GetAllRepositoryItems<BusinessFlow>())
             {
                 //update the BF target applications 
+                bf.StartDirtyTracking();
                 foreach (var activity in bf.Activities)
                 {
                     //donot check for TargetPlugins, only for TargetApplications 
 
-                    if (activity.TargetApplication.Equals(app.NameBeforeEdit))
+                    if (string.Equals(activity.TargetApplication, app.NameBeforeEdit))
                     {
                         activity.StartDirtyTracking();
                         activity.TargetApplication = app.AppName;
 
                         numOfAfectedItems++;
                     }
                 }
+
+                foreach (TargetApplication bfTargetApp in bf.TargetApplications.Where((targetApp)=>targetApp is TargetApplication))
+                {
+                    bfTargetApp.StartDirtyTracking();
```

Comment:
Dirty tracking should start only in case of change in app name

Reply:
Ok , will implement this!

Reply:
Implemented